lib/geocoder/lookups/olleh.rb

```diff
@@ -188,14 +210,14 @@ def query_url_params(query)
           s = [query.options[:"vx#{x}"], query.options[:"vy#{x}"]]
           hash.merge!({ "VX#{x}" => s[0], "VY#{x}" => s[1]}) unless s[0].nil? && s[1].nil?
         end
-      when "convert_coord"
+      when 'convert_coord'
         hash = {
           x: query.text.first,
           y: query.text.last,
           inCoordType: Olleh.coord_types[query.options[:coord_in]],
           outCoordType: Olleh.coord_types[query.options[:coord_out]]
        }
-      when "reverse_geocoding"
+      when 'reverse_geocoding'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.